app/controllers/chatfuel_controller.rb

```diff
@@ -53,6 +53,26 @@ def json_response
           }
         }
       ]
+    elsif @question_answer && !@next_question_answer
+      topic = @text_component.topic
+      messages =  [
+          {
+          attachment: {
+            payload: {
+              template_type: "button",
+              text: "#{content}",
+              buttons: [
+                {
+                  type: "show_block",
+                  block_name: "cont_" + topic.name,
+                  title: "Zurück",
+                }
+            ]
+            },
+            type: "template"
+          }
+        }
+      ]
     else
```

Comment:
I didn't use the else block because I think it is also used by the show function (which also calls json_response), if only a chatfuel topic is requested, without question_answers. So I figured I had to use a block only for the use case that there are question_answers and it is the last question_answer.